src/test/java/ch/admin/bag/covidcertificate/sdk/core/verifier/nationalrules/DisplayValidityCalculatorTest.kt

```diff
@@ -76,7 +76,7 @@ class DisplayValidityCalculatorTest {
 		)
 		assertNotNull(validityRange)
 		val validFrom = vaccinationDate.plusDays(21)
-		val validUntil = vaccinationDate.plusDays(365 + 21)
+		val validUntil = vaccinationDate.plusDays(269 + 21)
```

Comment:
Here 365 got replaced with 269 and further down 364 is also replaced with 269. Is that correct?